akka-actor-typed/src/main/resources/reference.conf

```diff
@@ -76,6 +76,9 @@ akka.reliable-delivery {
       # The ProducerController retries requests to the durable queue this
       # number of times before failing.
       retry-attempts = 10
+
+      # The ProducerController uses this interval as a delay between sending messages.
+      resend-interval = 1s
```

Comment:
we should name this `resend-first-interval` to avoid misunderstanding that the ProducerController would resend all messages with this interval

Reply:
and also change corresponding name in the Settings

Reply:
Done